.cirrus.yml

```diff
@@ -28,8 +28,8 @@ gcp_credentials: ENCRYPTED[d6efdb7d6d4c61e3831df2193ca6348bb02f26cd931695f69d419
 gce_instance:  &standard_vm
     image_project: "libpod-218412"
     zone: "us-central1-c"
-    cpu: 2
-    memory: "4Gb"
+    cpu: 8 
+    memory: "8Gb"
```

Comment:
Does this affect all jobs? I think we want to only set this for the e2e test to save some resources/money

Reply:
Yes

Reply:
k